### PR TITLE
Adding MonadA

### PR DESCRIPTION
**Pull request by Katochimoto**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3600,5 +3600,19 @@
...
+    "ticker": "MONA",
+    "poolId": "6a84654c217f1923b086e076192b50042911c7b7af3987a1992b77e9",
+    "social": {
+      "website": "https://monada.digital",
+      "twitter": "monadapool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MonadA",
```
Comment (Bez625) [inline registry.json:3614]: Please can you add a user TG handle? This must be the handle of a user and not a group or channel.

File: registry.json [hunk 2]
```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+  {
+    "ticker": "MONA",
+    "poolId": "6a84654c217f1923b086e076192b50042911c7b7af3987a1992b77e9",
```
Comment (Bez625) [inline registry.json:3606]: I checked your pool and it says your pool's pledge has not been met. I'm going to request you fix this before I approve if only to raise your awareness of the situation...

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 2 above)
Comment (Katochimoto) [reply registry.json:3662]: Had to wait for the end of an epoch

File: registry.json (hunk 1 above)
Comment (Katochimoto) [reply registry.json]: Now should be okay, thank you

**Pull request merged**